cmake/CMakeLists.txt

```diff
@@ -1,15 +1,18 @@
 # Minimum CMake required
 cmake_minimum_required(VERSION 2.8.12)
 
-# Project
-project(protobuf C CXX)
-
 # CMake policies
 cmake_policy(SET CMP0022 NEW)
 
+# Project
+project(protobuf C CXX)
+
 # Options
 option(protobuf_VERBOSE "Enable for verbose output" OFF)
+option(protobuf_TARGETS_ALIASES "Provide targets aliases without protobuf:: namespace" ON)
+option(protobuf_MODULE_COMPATIBLE "CMake build-in FindProtobuf.cmake module compatible" ON)
```

Comment:
IMO, if provided, this should default to off. Users can use FindProtobuf.cmake *OR* the .config files, but they cannot use both. Removing this does not remove a user's option to use the FindProtobuf.cmake file. Making the .config files support the syntax of the FindProtobuf module adds ongoing support burden with little real benefit.